wpilibc/athena/include/SensorBase.h

```diff
@@ -0,0 +1,52 @@
+/*----------------------------------------------------------------------------*/
```

Comment:
There is now differences between the Sim and Athena. This file was in shared, so it had to be split.